commit 6ef19ab7fa1535d35006535dba6c407dad2d845c
Author: Chen Gong <gong.chen@linux.intel.com>
Date:   Thu Mar 15 16:53:37 2012 +0800

    Update documentation for parameter *notrigger* in einj.txt
    
    Add description of parameter notrigger in the einj.txt.
    One can utilize this new parameter to do some SRAR injection
    test. Pay attention, the operation is highly depended on the
    BIOS implementation. If no proper BIOS supports it, even if
    enabling this parameter, expected result will not happen.
    
    v2:
      Update the documentation suggested by Tony
    
    Suggested-by: Tony Luck <tony.luck@intel.com>
    Signed-off-by: Chen Gong <gong.chen@linux.intel.com>
    Signed-off-by: Len Brown <len.brown@intel.com>

diff --git a/Documentation/acpi/apei/einj.txt b/Documentation/acpi/apei/einj.txt
index e7cc36397217..e20b6daaced4 100644
--- a/Documentation/acpi/apei/einj.txt
+++ b/Documentation/acpi/apei/einj.txt
@@ -53,6 +53,14 @@ directory apei/einj. The following files are provided.
   This file is used to set the second error parameter value. Effect of
   parameter depends on error_type specified.
 
+- notrigger
+  The EINJ mechanism is a two step process. First inject the error, then
+  perform some actions to trigger it. Setting "notrigger" to 1 skips the
+  trigger phase, which *may* allow the user to cause the error in some other
+  context by a simple access to the cpu, memory location, or device that is
+  the target of the error injection. Whether this actually works depends
+  on what operations the BIOS actually includes in the trigger phase.
+
 BIOS versions based in the ACPI 4.0 specification have limited options
 to control where the errors are injected.  Your BIOS may support an
 extension (enabled with the param_extension=1 module parameter, or